src/rdo.rs

```diff
@@ -288,14 +288,6 @@ pub fn rdo_mode_decision(
       if chroma_mode == PredictionMode::UV_CFL_PRED {
         if !best_mode_chroma.is_intra() { continue; }
         let mut wr: &mut dyn Writer = &mut WriterCounter::new();
-        // these rules follow TX_MODE_LARGEST
-        let tx_size = match bsize {
-            BlockSize::BLOCK_4X4 => TxSize::TX_4X4,
-            BlockSize::BLOCK_8X8 => TxSize::TX_8X8,
-            BlockSize::BLOCK_16X16 => TxSize::TX_16X16,
-            _ => TxSize::TX_32X32
-        };
-        let tx_type = TxType::DCT_DCT;
```

Comment:
[AWCY results](https://beta.arewecompressedyet.com/?job=rav1e-cfl-pred-full%402018-08-29T09%3A19%3A18.651Z&job=rav1e-cfl-pred-full-2%402018-08-29T09%3A51%3A13.330Z) for fixing this mistake:

|   PSNR | PSNR Cb | PSNR Cr | PSNR HVS |   SSIM | MS SSIM | CIEDE 2000 |
|   ---: |    ---: |    ---: |     ---: |   ---: |    ---: |       ---: |
| 0.0698 | -0.6611 | -0.2755 |   0.1140 | 0.0975 |  0.1082 |    -0.1782 |